src/libData/AccountData/Account.h

```diff
@@ -43,25 +43,23 @@
 /// DB storing trie storage information for all accounts.
 // static OverlayDB contractStatesDB("contractStates");
 
+static uint32_t scilla_version_place_holder;
```

Comment:
This variable used without initialize?

Reply:
this is used as place holder, simply used as default reference argument, don't think we need to initialize it